src/renderer/dx/CustomTextRenderer.cpp

```diff
@@ -488,28 +492,39 @@ using namespace Microsoft::Console::Render;
     ::Microsoft::WRL::ComPtr<ID2D1DeviceContext> d2dContext;
     RETURN_IF_FAILED(clientDrawingContext->renderTarget->QueryInterface(d2dContext.GetAddressOf()));
 
-    // Using the context is the easiest/default way of drawing.
-    d2dContext->DrawGlyphRun(baselineOrigin, glyphRun, glyphRunDescription, brush, measuringMode);
+    // If a special drawing effect was specified, see if we know how to deal with it.
+    if (clientDrawingEffect)
+    {
+        ::Microsoft::WRL::ComPtr<IBoxDrawingEffect> boxEffect;
+        if (SUCCEEDED(clientDrawingEffect->QueryInterface<IBoxDrawingEffect>(&boxEffect)))
+        {
+            return _DrawBoxRunManually(clientDrawingContext, baselineOrigin, measuringMode, glyphRun, glyphRunDescription, boxEffect.Get());
+        }
 
-    // However, we could probably add options here and switch out to one of these other drawing methods (making it
-    // conditional based on the IUnknown* clientDrawingEffect or on some other switches and try these out instead:
+        //_DrawBasicGlyphRunManually(clientDrawingContext, baselineOrigin, measuringMode, glyphRun, glyphRunDescription);
+        //_DrawGlowGlyphRun(clientDrawingContext, baselineOrigin, measuringMode, glyphRun, glyphRunDescription);

```

Comment:
Stray comment?

Reply:
kept on purpose

Reply:
Yes that's there on purpose to remind me how to add more things later. Sorry. It's been there a while, it just moved positions.